OpenRA.Mods.Common/Activities/FindResources.cs

```diff
@@ -123,6 +124,17 @@ public override Activity Tick(Actor self)
 			return Util.SequenceActivities(mobile.MoveTo(path[0], 1), new HarvestResource(), new FindResources());
 		}
 
+		static int EstimateDistance(CPos here, CPos destination)
+		{
+			var diag = Math.Min(Math.Abs(here.X - destination.X), Math.Abs(here.Y - destination.Y));
+			var straight = Math.Abs(here.X - destination.X) + Math.Abs(here.Y - destination.Y);
+
+			// According to the information link, this is the shape of the function.
```

Comment:
Sorry, it's a copy and paste of the Estimation Heuristic from the pathfinder. I'll remove that comment.

In a future the heuristics functions should be reestructured to avoid this kind of duplication.